src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
```

Comment:
The width of the calendar should be calculated based on the size of the day cell and the gap between them. Using a fixed width like `706px` is not flexible and may not work well if the size of the day cells or the gap changes.

Reply:
Já alterei